packages/trader/src/Modules/Trading/Components/Form/ContractType/contract-type-widget.jsx

```diff
@@ -205,6 +205,7 @@ const ContractTypeWidget = ({ is_equal, name, value, list, onChange, languageCha
 
 ContractTypeWidget.propTypes = {
     is_equal: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+    languageChanged: PropTypes.bool,
```

Comment:
Shouldn't it be `is_language_changed`?

Reply:
They defined the property name like this